ext/opentelemetry-ext-zipkin/src/opentelemetry/ext/zipkin/__init__.py

```diff
@@ -212,6 +212,17 @@ def _extract_tags_from_span(attr):
     return tags
 
 
+def _extract_tags_from_span(span: Span):
+    tags = _extract_tags_from_dict(span.attributes)
+    if span.resource:
+        resource_tags = _extract_tags_from_dict(span.resource.labels)
+        if tags:
```

Comment:
you can eliminate this check if _extract_tags_from_dict just returns back the empty dict. Also this would also be simplified by span attributes returning an empty dictionary anyway.

Reply:
done